internal/rego/rego.go

```diff
@@ -3,57 +3,53 @@ package rego
 import (
 	"fmt"
 	"io/ioutil"
+	"os"
+	"path/filepath"
 	"regexp"
 	"sort"
 	"strings"
 
 	"github.com/open-policy-agent/opa/ast"
 )
 
-// File is a parsed Rego file
+// File is a parsed Rego file.
 type File struct {
 	FilePath       string
 	PackageName    string
 	ImportPackages []string
 	Contents       string
-	RulesActions   []string
+	RuleNames      []string
 	Comments       []string
 }
 
-// LoadPoliciesWithAction loads all policies from rego with rules with a given action name
-func LoadPoliciesWithAction(files []string, action string) ([]File, error) {
-	regoFiles, err := loadRegoFiles(files)
+// GetFiles gets all Rego files in the given path and its subdirectories.
+func GetFiles(path string) ([]File, error) {
+	filePaths, err := getFilePaths(path)
 	if err != nil {
-		return nil, fmt.Errorf("load rego files: %w", err)
+		return nil, fmt.Errorf("load files: %w", err)
 	}
 
-	policies := getPoliciesWithAction(regoFiles, action)
-	return policies, nil
-}
-
-// LoadPolicies loads all policies from rego with rules
-func LoadPolicies(files []string) ([]File, error) {
-	regoFiles, err := loadRegoFiles(files)
+	files, err := getFiles(filePaths)
 	if err != nil {
-		return nil, fmt.Errorf("load rego files: %w", err)
+		return nil, fmt.Errorf("load files: %w", err)
 	}
 
-	policies := getPolicies(regoFiles)
-	return policies, nil
+	return files, nil
 }
 
-// LoadLibraries loads all libraries from rego
-func LoadLibraries(files []string) ([]File, error) {
-	regoFiles, err := loadRegoFiles(files)
+// GetFilesWithAction gets all Rego files in the given path and its subdirectories that contain the specified action.
+func GetFilesWithAction(path string, action string) ([]File, error) {
```

Comment:
If we are calling them RuleNames in the struct, this should probably be `GetFilesWithRuleNames` or similar